sub-packages/bionemo-esm2/src/bionemo/esm2/scripts/train_esm2.py

```diff
@@ -163,10 +164,17 @@ def main(
     strategy = nl.MegatronStrategy(
         tensor_model_parallel_size=tensor_model_parallel_size,
         pipeline_model_parallel_size=pipeline_model_parallel_size,
-        ddp="megatron",
+        pipeline_dtype=get_autocast_dtype(precision),
+        ddp=DistributedDataParallelConfig(
+            check_for_nan_in_grad=True,
+            overlap_grad_reduce=True,
+            overlap_param_gather=True,
```

Comment:
Does this make us possibly exposed to the loss spike NeMo bug where they saw issues with overlap_param_gather?

Reply:
Good call. Do they have a PR merged?

Reply:
Took it off for the moment.